### ISSUE TITLE
ER: update the pr template

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#6058]
### Emergent Requirement - Problem
We have observed non-members creating PRs. To solve this, we will add a notice to the top of the the PR template that notifies developers that they must complete onboarding before creating a PR. 

### Issue you discovered this emergent requirement in
- #3906

### Date discovered 
2024-01-03

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@ExperimentsInHonesty 

### What happens if this is not addressed
Non-members will continue to create PRs before completing onboarding.

### Resources
None

### Recommended Action Items
- [x] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft]
Add the following text to add to to top of [pull_request_template.md](https://github.com/hackforla/website/blob/gh-pages/.github/pull_request_template.md):
```
Please note: You must be a member of the HFLA  website team in order to create pull requests. Please see our page on how to join us as a member at HFLA: https://www.hackforla.org/getting-started.   Delete this message if you joined this team via onboarding.
```

When you make the issue, you will need to include some information on how to test the template.

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge Can someone from the merge team review this ER and if you think it is ready for issue making, please add a `ready for prioritization` label.

**Comment by w1ld-r** [comment]
@bonniewolfe Reviewing this now. Just to clarify, is this ER intending to just add the "you must be a member" text to the PR template? And no GHA improvements are to be included in the issue?

**Comment by ExperimentsInHonesty** [comment]
for now, just the notice.  If in the future the problem is not solved by the notice, then we will write a GHA.

**Comment by HackforLABot** [comment]
Hi @siyunfeng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by siyunfeng** [comment]
Availability: 6-10 PM Weekdays (EDT)
ETA: 11/17/2024 Sunday

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty @siyunfeng   

I was pinged when this ER was assigned.  Since the time of Bonnie's note above...

> for now, just the notice. If in the future the problem is not solved by the notice, then we will write a GHA.

…we do have a functioning GHA that prevents non-members from submitting PRs:
- it was completed by @ajb176 (thanks!) here: #6971 
- and here it is in action: #7653  

This is only an FYI: we may still want to include the notice shown on this ER on the PR template.

**Comment by siyunfeng** [comment]
> @ExperimentsInHonesty @siyunfeng
> 
> I was pinged when this ER was assigned. Since the time of Bonnie's note above...
> 
> > for now, just the notice. If in the future the problem is not solved by the notice, then we will write a GHA.
> 
> …we do have a functioning GHA that prevents non-members from submitting PRs:
> 
> * it was completed by @ajb176 (thanks!) here: [Verify that PR creator is a member of the website-write team #6971](https://github.com/hackforla/website/pull/6971)
> * and here it is in action: [chore: LA TDM Calculator project profile update #7653](https://github.com/hackforla/website/pull/7653)
> 
> This is only an FYI: we may still want to include the notice shown on this ER on the PR template.

Hi @t-will-gillis , I also noticed that the related issue to make a GHA was created and resolved. 

Bonnie required me to work on creating this issue at the meeting on Tuesday. I went through this ER and did not see the notice in the PR template, so I assigned myself to work on it. 

@ExperimentsInHonesty , should I continue working on creating an issue to add the notice in the PR template?

**Comment by siyunfeng** [comment]
Hi @ExperimentsInHonesty , @t-will-gillis ,

I created a new issue for this ER: 
- #7722

I added `Complexity: Small`, `role: back end/devOps`, and `Feature: Board/GitHub Maintenance` to the issue. 
Please correct me if I need to change them, and what should I add to the label for **size**?

**Comment by siyunfeng** [comment]
### Provide Update

1. Progress: Created the issue based on this ER
2. Blockers: I'm still waiting for the merge team to review and approve issue #7722 
3. Availability: 6-10 PM Weekdays (EDT)
4. ETA: 11/24/2024 Sunday (once the issue is approved)

**Comment by roslynwythe** [comment]
- @siyunfeng thank you for writing #7722.  However I think that the instructions to test the PR in the fork of the dev's fork (" Follow these steps to test the updated PR template: ") should come before creating the final PR.

**Comment by siyunfeng** [comment]
> * @siyunfeng thank you for writing [Update PR Template with adding a notice of completing onboarding before creating a PR #7722](https://github.com/hackforla/website/issues/7722).  However I think that the instructions to test the PR in the fork of the dev's fork (" Follow these steps to test the updated PR template: ") should come before creating the final PR.

Hi @roslynwythe , thank you for reviewing the issue #7722 . I just updated the order of the instructions. 
Can you review the issue again? Thanks.
